www/profile/inner.js

```diff
@@ -158,8 +175,11 @@ define([
         }, Messages.profile_addLink);
         APP.$linkEdit = $(button);
         $block.append(button);
-        var save = h('button.btn.btn-primary', Messages.settings_save);
-        var text = h('input');
+        var save = h('button.btn.btn-primary', Messages.settings_save, { 'aria-labelledby': 'cp-save-link' });
```

Comment:
We usually put the attribute object before the content. This order should work but it is a bit inconsistent with the rest of the code.